rotkehlchen/tests/unit/test_defi_oracles.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from rotkehlchen.constants.assets import A_1INCH, A_AAVE, A_BTC, A_DOGE, A_ETH, A_WETH
+from rotkehlchen.constants.misc import ONE, ZERO
+from rotkehlchen.errors import PriceQueryUnsupportedAsset
+from rotkehlchen.inquirer import CurrentPriceOracle
+from rotkehlchen.types import Price
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+def test_uniswap_oracles_asset_to_asset(inquirer_defi):
```

Comment:
Since you also test the case of non-ethereum tokens and using ETH as an intermediary you have to test that.

you also have to make sure to use both uniswap v3 and v2 in tests. And also at least one test that uses more than one hop pool